README.md

```diff
@@ -3,6 +3,14 @@
 Example content for demonstrating the LocalGov Drupal distribution and to help
 with development.
 
+Enabling this module through the admin interface is not supported. Attempting to enable this module through the admin interface will likely fail as it exceeds PHP execution limits.
+
+Instead, install the module from the terminal using
+
+```bash
+drush en localgov_demo
+```
```

Comment:
We might need to add something here for lando / ddev users?
so 
```bash
drush en localgov_demo
```
If you use Lando / DDev, add the relevant app command before.
```bash
lando drush en localgov_demo
```

Reply:
OK, have updated.